commit 95e42e9944f70640ec4d7a2ea4973401700bcd6b (refs/changes/62/2762/2)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2017-10-10T15:05:06-07:00 (2 years ago)
    
    NAPI-433 listNetworkIPs schema should accept owner_uuid (Doc typo)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/docs/index.md b/docs/index.md
index b7d7974..c2d9318 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -687,7 +687,7 @@ List VLANs owned by a user.
 
 Create a new fabric VLAN.
 
-## Inputs
+### Inputs
 
 | Field              | Type             | Description                                                                            |
 | ------------------ | ---------------- | -------------------------------------------------------------------------------------- |
